### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by mchompalova**
# Description

## What

Add runbook for the service.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-1181

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [x] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by bboykov — DISMISSED** [review]

**Review by bboykov — APPROVED** [review]

**Review by bboykov — DISMISSED** [review]

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
+- <https://upp-prod-delivery-eu.upp.ft.com/__health>
+- <https://upp-prod-delivery-us.upp.ft.com/__health>
```
Comment (karsov) [inline runbooks/runbook.md:102]: In other services we provide direct links to the service healthchecks.

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+The service is deployed in the delivery clusters.
```
Comment (karsov) [inline runbooks/runbook.md:67]: In other runbooks here we have a link to the failover docs.

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# UPP - Public Suggestions API
```
Comment (karsov) [inline runbooks/runbook.md:1]: The dependences are missing. Some (not all) of them a listed in the current runbook.

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+A service serving requests made towards the suggestions umbrella.
+
+## Code
+
+public-suggestions-api
+
+## Primary URL
+
+<https://github.com/Financial-Times/public-suggestions-api>
```
Comment (karsov) [inline runbooks/runbook.md:11]: The service has a public URL. It is referenced in the current runbook.

**Review by karsov — CHANGES_REQUESTED** [review]
I was surprised that there are a lot more changes in this PR other than a runbook.

I have reviewed the runbook only for the moment.

**Review by bboykov — DISMISSED** [review]

**Review by bboykov — APPROVED** [review]

File: service/conceptFilter.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+	baseURL       string
+	endpoint      string
+	client        Client
+	systemID      string
+	name          string
+	failureImpact string
+
+	deniedUUIDs []string
+	uuidsMx     *sync.RWMutex
+	refreshing  bool
```
Comment (karsov) [inline service/conceptFilter.go:25]: This property is not used.

File: service/conceptFilter.go [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+		if strings.Contains(conceptID, uuid) {
+			return false
+		}
+	}
+	return true
+}
+
+func (f *CachedConceptFilter) RefreshCache(ctx context.Context, tid string) error {
+	f.uuidsMx.Lock()
+	defer f.uuidsMx.Unlock()
```
Comment (karsov) [inline service/conceptFilter.go:54]: This means that during the whole HTTP request `IsConceptAllowed ` will be blocked and wait for the cache refresh.

**Review by karsov — CHANGES_REQUESTED** [review]
Let's separate the runbook in another PR.
I don't have enough context for the code changes and also they look problematic to me.

**Review by ekaneva — COMMENTED** [review]
Please rename this PR as it is misleading. The runbook was already approved and merged through a separate PR. This one consist of changes that should be reviewed within the Metadata team in my opinion.

**Pull request closed without merging**